arangod/Agency/Node.cpp

```diff
@@ -1219,6 +1219,18 @@ void Node::clear() {
   }
 }
 
+std::vector<std::string> const Node::keys() const {
```

Comment:
the `const` probably does not have any effect here.
```suggestion
std::vector<std::string> Node::keys() const {
```